lib/linguist/languages.yml

```diff
@@ -3322,6 +3322,13 @@ ObjDump:
   tm_scope: objdump.x86asm
   ace_mode: assembly_x86
   language_id: 256
+ObjectScript:
+  type: programming
+  extensions:
+  - ".cls"
+  language_id: 202735509
+  color: "#424893"
+  ace_mode: text
```

Comment:
You're missing the following field:

~~~yml
tm_scope: source.objectscript
~~~

This is what specifies the grammar to use for syntax highlighting.